tools/dashboard/probot/test/payloads/files/files.unrelated.json

```diff
@@ -1,50 +1,50 @@
 [
```

Comment:
Why is this file touched by this PR?

Reply:
@RandellDawson I should have asked, because I am not sure of this files purpose, but it makes/made reference to:
```
...,\n-  isPrivate: Joi.bool(),\n-  isRequired: Joi.bool(),\n-  name: Joi.string(),\n-  
```
In the `"patch"` property

Reply:
Oh I see now.  It would not hurt anything to remove it or keep it.  That is a fixture that represents a pull request showing the patch for the changes being made.  It just so happens that the PR example in the fixture had that code in it.